modules/repository-azure/build.gradle

```diff
@@ -75,7 +75,7 @@ dependencies {
   api "org.ow2.asm:asm:9.3"
 
   runtimeOnly "com.google.crypto.tink:tink:1.14.0"
-  runtimeOnly "com.google.protobuf:protobuf-java:4.27.0"
+  runtimeOnly "com.google.protobuf:protobuf-java:4.27.5"
```

Comment:
Wondering if we should align protobuf dependency and use `versions.protobuf` here as we did with in other modules?